third_party/tsl/third_party/mkl_dnn/mkldnn_v1.BUILD

```diff
@@ -6,12 +6,15 @@ exports_files(["LICENSE"])
 
 _CMAKE_COMMON_LIST = {
     "#cmakedefine DNNL_GPU_RUNTIME DNNL_RUNTIME_${DNNL_GPU_RUNTIME}": "#define DNNL_GPU_RUNTIME DNNL_RUNTIME_NONE",
+    "#cmakedefine DNNL_GPU_VENDOR DNNL_VENDOR_${DNNL_GPU_VENDOR}": "#define DNNL_VENDOR_INTEL",
```

Comment:
Intel GPU changes are in the PJRT plug-in. Do we need this configuration here?

Reply:
I mean, we can set it to `DNNL_VENDOR_NONE`, right?

Reply:
That should work as well, at least I don't anticipate any issues why it shouldn't.

Reply:
Yes, you can set it to DNNL_VENDOR_NONE, do you want me to change it?

Reply:
Yes, please. Thank you! :)